README.adoc

```diff
@@ -40,6 +40,44 @@ There are a few more targets that you can find useful:
 * to remove the local test namespace: `$ make clean-namespace`
 * to remove & create the local test namespace, and create ClusterRole/ClusterRoleBinding and ServiceAccount inside of the namespace: `$ make reset-namespace`
 
+== Installing operator
+
+To install the host operator via OperatorHub you need to have OpenShift 4.2+ running and access to a docker registry.
+Since the operator is no available in OperatorHub nor in any registry by default, you need to deploy the image to the docker registry, create `CatalogSource` and `ClusterServiceVersion` in the `openshift-marketplace` namespace and then create `OperatorGroup` and `Subscription` in the namespace you want to install the operator to.
+Before running any make target, make sure that you have `QUAY_NAMESPACE` variable set to your quay username (or set to any namespace you want to push the image to).
+
+```bash
+$ export QUAY_NAMESPACE=<quay-username>
+```
+
+Then, to install the operator run:
+```bash
+$ make install-operator
+```
+
```

Comment:
There was one more step required for me. `make install-operator` created `host-operator` repository in my quay.io but after that I had to go to quay.io configuration and made that repo public (it was created as private). Until then OS could not pull the image.
Let's mention this step here too.